src/index.html

```diff
@@ -1,12 +1,351 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="/images/logo.png" alt="Logo">
+      </a>
+      <nav class="main-navigation">
+        <ul class="main-navigation__list">
+          <li class="main-navigation__item"><a href="#apple" class="is-active">Apple</a></li>
+          <li class="main-navigation__item"><a href="#samsung">Samsung</a></li>
+          <li class="main-navigation__item"><a href="#smartphones">Smartphones</a></li>
+          <li class="main-navigation__item">
+            <a href="#laptops_computers" data-qa="nav-hover">Laptops & Computers</a>
+          </li>
+          <li class="main-navigation__item"><a href="#gadgets">Gadgets</a></li>
+          <li class="main-navigation__item"><a href="#tablets">Tablets</a></li>
+          <li class="main-navigation__item"><a href="#photo">Photo</a></li>
+          <li class="main-navigation__item"><a href="#video">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-page">
+      <div class="container">
+        <div class="card" data-qa="card">
+          <img
+          class="card__image"
+          src="images/imac.jpeg"
+          alt="iMac" height="134px">
```

Comment:
Quickly review the [styling guide](https://mate-academy.github.io/style-guides/htmlcss.html#good)